src/coordinator/api/v1/handler/namespace/delete.go

```diff
@@ -38,10 +38,13 @@ import (
 
 const (
 	namespaceIDVar = "id"
+
+	// DeleteHTTPMethod is the HTTP method used with this resource.
+	DeleteHTTPMethod = "DELETE"
```

Comment:
hm would it be better to create an enum for this type - GET/DELETE/POST/PUT/...  ?